translations/LatamSpanish/docs.md

```diff
@@ -0,0 +1,18 @@
+Latin American Spanish is more formal than Spain Spanish.
+
+For example: Latin American varieties of Spanish do not use vosotros (you, plural, informal), it is preferred the formal ustedes.
+
+The vast majority of Spanish words are universal, but some are not.
+Some examples such as: teléfono móvil / celular and ordenador / computadora,
+with the second of each pair being the Latin American form.
+
+
+Below are some words commonly used in applications:
+
+Home : Inicio
```

Comment:
These lines are joined in the rendered view:
![screen shot 2017-01-16 at 10 17 12 am](https://cloud.githubusercontent.com/assets/641763/21971718/0a198fd6-dbd5-11e6-9bc8-7edb99a7ffa6.png)

See [here](https://github.com/Juanvulcano/zulip-gci/blob/492e29bcc6b382209bcc39667af223169dc60230/translations/LatamSpanish/docs.md).